src/main/java/justbot/command/UnmarkCommand.java

```diff
@@ -0,0 +1,58 @@
+package justbot.command;
+
+import justbot.exception.JustbotException;
+import justbot.storage.Storage;
+import justbot.task.Task;
+import justbot.task.TaskList;
+import justbot.ui.Ui;
+
+/**
+ * Represents a command to unmark a task as not done in the Justbot application.
+ * The UnmarkCommand marks a specific task in the task list as not completed.
+ */
+public class UnmarkCommand extends Command {
+    private int unmarkNumber;
+
+    /**
+     * Constructs an UnmarkCommand with the specified task number to unmark as not done.
+     *
+     * @param unmarkNumber The 1-based index of the task to be unmarked as not done.
+     */
+    public UnmarkCommand(int unmarkNumber) {
```

Comment:
As with the other comments mentioned above, I feel that taskNumberToUnmark is more appropriate as it is more descriptive